setup.py

```diff
@@ -58,12 +58,15 @@
             "Please install amdsmi from https://github.com/ROCm/amdsmi to enable this feature."
         )
 
+
+
 if USE_ROCM:
     AUTOAWQ = "autoawq@https://github.com/casper-hansen/AutoAWQ/releases/download/v0.2.1/autoawq-0.2.1+rocm571-cp310-cp310-linux_x86_64.whl"
     AUTOGPTQ = "auto-gptq@https://huggingface.github.io/autogptq-index/whl/rocm573/auto-gptq/auto_gptq-0.7.1%2Brocm5.7.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl"
+
 else:
-    AUTOAWQ = "autoawq==0.2.1"
-    AUTOGPTQ = "auto-gptq==0.7.1"
+    # AUTOAWQ will be installed from source via scripts/install_autoawq.py script
+    AUTOGPTQ = "auto-gptq@git+https://github.com/PanQiWei/AutoGPTQ.git"
```

Comment:
we can't have this kind of links in the setup unfortunately because pypi doesn't accept them.
so yeah what I would suggest is to have normal pypi libraries in setup, and do a warning if gptq/awq kernels are not importable; as easy as trying to load the kernels directly `awq_ext`, `exl2_ext`, etc. And directing to the installation from source scripts.

Reply:
Unfortunately, I cannot directly check for installed packages within setup.py (without --no-build-isolation). An alternative approach could be to throw a runtime error if the required kernels are not available when attempting to use them. After the error, we could provide a warning to an installation script that users can run to install both libraries from source. What do you think of this approach?

Reply:
yes that's what I meant, a runtime warning, we keep setup as clean as possible.

Reply:
I’ve added a runtime check for the kernels and extended the CLI with new commands for installing autoAWQ and autoGPTQ from source, making it more convenient for users. The commands are optimum-benchmark +install_auto_awq_from_source=True and optimum-benchmark +install_auto_gptq_from_source=True. Unfortunately, I couldn’t make the arguments simpler (like --install_auto_awq) due to limitations with the Hydra configuration

Reply:
I'm against adding this complexity to the cli command `optimum-benchmark`. let's leave them as scripts in the scripts folder, that can be used after cloning the repo (from source).

Reply:
Ok, I revert them back to scripts folder, so they can be run after cloning the repo. I also added this to the documentation `curl -s https://raw.githubusercontent.com/huggingface/optimum-benchmark/main/scripts/install_quantization_libs.py | python - --install-autoawq-from-source --install-autogptq-from-source` so that the user can install them more easily